src/languageservice/httpClient.ts

```diff
@@ -77,11 +78,13 @@ export default class HttpClient implements IHttpClient {
         };
 
         if (url.protocol === 'https:') {
+            let caList = getTrustedCertificates();
```

Comment:
we should put this behind a user setting.  I'm not sure if the default should be on or off?